MetaMask/metamask-extension
# Adding new icons

## Explanation
Adding new icons provided in #18040

List of icons
- Twitter
- QrCode
- UserCheck
- Ban
- Bold
- CircleX
- Download
- File
- Flask
- Plug
- Share
- Square
- Tint
- Upload
- Usb
- Wifi

* Fixes #18040

## Screenshots/Screencaps

### After

<img width="200" alt="Screenshot 2023-04-28 at 5 51 05 PM" src="https://user-images.githubusercontent.com/8112138/235095624-66de3bdc-7cd1-476b-8dda-9afa06e0b42b.png"><img width="200" alt="Screenshot 2023-04-28 at 5 51 10 PM" src="https://user-images.githubusercontent.com/8112138/235095634-896a4dad-ac3c-4744-88eb-8ac2a9a7fb65.png"><img width="200" alt="Screenshot 2023-04-28 at 5 51 22 PM" src="https://user-images.githubusercontent.com/8112138/235095637-f5364520-cd34-4a9a-b338-b40af04c3172.png"><img width="200" alt="Screenshot 2023-04-28 at 5 51 30 PM" src="https://user-images.githubusercontent.com/8112138/235095646-cea190d5-752f-4022-9cb0-b65902f8f0fe.png"><img width="200" alt="Screenshot 2023-04-28 at 5 51 44 PM" src="https://user-images.githubusercontent.com/8112138/235095653-f6600814-9264-4374-b5b5-621dcb2561d7.png"><img width="200" alt="Screenshot 2023-04-28 at 5 51 50 PM" src="https://user-images.githubusercontent.com/8112138/235095659-a451fa7e-c43f-41fd-bc93-6c2efa5c64ac.png"><img width="200" alt="Screenshot 2023-04-28 at 5 51 55 PM" src="https://user-images.githubusercontent.com/8112138/235095665-52801ba7-a8c9-47ce-abc2-457f3dbc549c.png"><img width="200" alt="Screenshot 2023-04-28 at 5 52 01 PM" src="https://user-images.githubusercontent.com/8112138/235095667-bb642d64-81af-44c9-bc01-98c0e4ba1cd4.png"><img width="200" alt="Screenshot 2023-04-28 at 5 52 07 PM" src="https://user-images.githubusercontent.com/8112138/235095670-6a156d22-73f9-4725-a60f-01c6b1311a63.png"><img width="200" alt="Screenshot 2023-04-28 at 5 52 13 PM" src="https://user-images.githubusercontent.com/8112138/235095675-0141b4a2-2bc8-4f22-b2ea-4ee732415505.png"><img width="200" alt="Screenshot 2023-04-28 at 5 52 18 PM" src="https://user-images.githubusercontent.com/8112138/235095679-97ac600a-f201-4ad3-bc97-f862b1b30561.png"><img width="200" alt="Screenshot 2023-04-28 at 5 52 31 PM" src="https://user-images.githubusercontent.com/8112138/235095686-1ec10328-0e01-4623-9498-6de4d8528f2a.png"><img width="200" alt="Screenshot 2023-04-28 at 5 52 50 PM" src="https://user-images.githubusercontent.com/8112138/235095691-f88ab12c-e3aa-48f4-8ffe-91dbdc299788.png"><img width="200" alt="Screenshot 2023-04-28 at 5 53 06 PM" src="https://user-images.githubusercontent.com/8112138/235095693-94c1a690-687c-4a8e-b704-a6c3218c1f9b.png"><img width="200" alt="Screenshot 2023-04-28 at 5 53 12 PM" src="https://user-images.githubusercontent.com/8112138/235095696-b6814431-1ead-4f92-84e5-fe30d0db4931.png"><img width="200" alt="Screenshot 2023-04-28 at 5 53 19 PM" src="https://user-images.githubusercontent.com/8112138/235095704-f0aee021-1ed9-4481-b260-58146b6a0d97.png">


## Manual Testing Steps
- Go to the latest build of storybook in this PR
- Search `Icon` in the search panel
- Using the icon search check all icons are present and accounted for

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
